platform-modules/batch-models/src/main/scala/org/ekstep/analytics/updater/UpdateWorkFlowUsageDB.scala

```diff
@@ -31,6 +31,7 @@ object UpdateWorkFlowUsageDB extends IBatchModelTemplate[DerivedEvent, DerivedEv
             val summType = x.dimensions.`type`.get
             val did = x.dimensions.did.get
             val contentId = x.dimensions.content_id.get;
+            val contentType = x.dimensions.content_type.getOrElse("")
```

Comment:
The content_type is already an Option. We don't have to add a default value and convert it to an Option again.